[18.0][MIG] mrp_production_auto_validate: Migration to 18.0

chaule97 opened the pull request:
- I think we don't need field `mo_auto_validation_warning`, we only show the warning `<div/>` tag when `mo_auto_validation = True`
- `mrp.immediate.production.line` was [removed](https://github.com/odoo/odoo/pull/108446), we have to handle `mrp.production.backorder` instead

oihane committed:
[MOD] Added <mrp_partner_note> and renamed *notes modules to *note

pedrobaeza committed:
[IMP] Expand AUTHORS

oihane committed:
[MOD] Changed email

oihane committed:
[IMP] <mrp_bom_note> Added OCA required README and info on __openerp__.py

oihane committed:
[IMP] <mrp_bom_note> Updated README

sbidoul committed:
[UPD] prefix versions with 8.0

pedrobaeza committed:
[MIG] Make modules uninstallable

oihane committed:
[MIG] Make mrp_bom_note installable in v9

Commit:
[ADD] mrp_sale_info

Commit:
[MIG] Make mrp_sale_info installable on v9

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Rename manifest files

bimajatiwijaya committed:
[10.0] [MIG] mrp_bom_note (#202)

Squashed commits:

OCA Transbot updated translations from Transifex

agung-rachamatullah committed:
[MIG] mrp_bom_note: Migration to 11.0

Squashed commit:

[UPD] Update mrp_bom_note.pot

[ADD] icon.png

oihane committed:
[MIG] mrp_sale_info: Migration to 11.0

marylla committed:
Translated using Weblate (German)

Currently translated at 100.0% (6 of 6 strings)

Translation: manufacture-11.0/manufacture-11.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-11-0/manufacture-11-0-mrp_sale_info/de/

skukered committed:
[MIG] mrp_sale_info: Migration to 12.0

MiquelRForgeFlow committed:
[ADD] mrp_bom_tracking

LoisRForgeFlow committed:
[12.0][MIG] mrp_bom_tracking

marylla committed:
Added translation using Weblate (German)

LoisRForgeFlow committed:
[12.0][ADD] mrp_multi_level_estimate

oca-travis committed:
[UPD] Update mrp_multi_level_estimate.pot

LoisRForgeFlow committed:
[12.0][FIX] mrp_multi_level_estimate: adapt to latest split of stock_demand_estimate module.

LoisRForgeFlow committed:
[12.0][UPD] mrp_multi_level_estimate: eficent rebranded to forgeflow.

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mrp_multi_level_estimate 12.0.1.1.0

JoanSForgeFlow committed:
[IMP] mrp_bom_tracking: black, isort

JoanSForgeFlow committed:
[MIG] mrp_bom_tracking: Migration to 13.0

lbelorgey committed:
[MIG] mrp_bom_note: Migration to 12.0

Squashed commits:

[UPD] Update mrp_bom_note.pot

[UPD] README.rst

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-12.0/manufacture-12.0-mrp_bom_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-12-0/manufacture-12-0-mrp_bom_note/ (+1 squashed commits)

ps-tubtim committed:
[IMP] mrp_bom_note: black, isort

ps-tubtim committed:
[IMP] mrp_multi_level_estimate: black, isort

ps-tubtim committed:
[MIG] mrp_multi_level_estimate: Migration to 13.0

oca-travis committed:
[UPD] Update mrp_multi_level_estimate.pot

pedrobaeza committed:
[FIX+IMP] mrp_sale_info: Isolate procurement group + tests

Commit:
[12.0][ADD] mrp_subcontracting_purchase_link

ps-tubtim committed:
[MIG] mrp_bom_note: Migration to 13.0

Squashed commits:

[UPD] Update mrp_bom_note.pot

oca-travis committed:
[UPD] Update mrp_subcontracting_purchase_link.pot

OCA-git-bot committed:
[ADD] icon.png

clementmbr committed:
[MIG] mrp_subcontracting_purchase_link: Migration from 12.0 to 14.0

Mat-moran committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-13.0/manufacture-13.0-mrp_bom_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-13-0/manufacture-13-0-mrp_bom_note/es/

DavidBForgeFlow committed:
[MIG] mrp_multi_level_estimate: Migration to 14.0

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 44.4% (4 of 9 strings)

Translation: manufacture-12.0/manufacture-12.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-12-0/manufacture-12-0-mrp_sale_info/pt_BR/

marcelsavegnago committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (9 of 9 strings)

Translation: manufacture-12.0/manufacture-12.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-12-0/manufacture-12-0-mrp_sale_info/pt_BR/

oca-travis committed:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
[UPD] README.rst

oca-travis committed:
[UPD] Update mrp_subcontracting_purchase_link.pot

OCA-git-bot committed:
[UPD] README.rst

kevinkhao committed:
[IMP] mrp_sale_info: black, isort, prettier

kevinkhao committed:
[MIG] mrp_sale_info: Migration to 14.0

tafaRU committed:
[IMP] mrp_sale_info: add Customer Reference to Manufacturing Order and Work Order

Commit:
[14.0][IMP] mrp_multi_level_estimate:do not require date ranges on stock demand estimates

BernatPForgeFlow committed:
[MIG] mrp_bom_tracking: Migration to 14.0

BernatPForgeFlow committed:
[MIG] mrp_bom_tracking: Migration to 15.0

OCA-git-bot committed:
mrp_multi_level_estimate 14.0.1.0.1

Commit:
[FIX] Fix access right issues when using actions

OCA-git-bot committed:
mrp_subcontracting_purchase_link 14.0.1.0.1

LoisRForgeFlow committed:
[FIX] mrp_multi_level_estimate: do not overstate estimates

when group days of estimates is greater than the lengh of the date
range in the estimate, the estimate is overstated and can generate
unexpected result. Enhance help message to highligh this and
prevent it from happening.

LoisRForgeFlow committed:
[FIX] mrp_multi_level_estimate: rounding issue

Rounding should be done at the end of the calculation, using the daily
quantity already rounded can lead to errors.

oca-travis committed:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
mrp_multi_level_estimate 14.0.1.0.2

LoisRForgeFlow committed:
[15.0][MIG] mrp_multi_level_estimate

Commit:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
[UPD] README.rst

BlommaertsEdwin committed:
[15.0][MIG] mrp_sale_info: Migration to 15.0

ChrisOForgeFlow committed:
[FIX] mrp_bom_tracking: product.uom -> uom.uom

LoisRForgeFlow committed:
[FIX] mrp_multi_level_estimate: group_estimate_days smaller than date range.

Do not overstate demand, a test hightlights the issue being solved.

OCA-git-bot committed:
mrp_multi_level_estimate 15.0.1.0.1

Commit:
[ADD] mrp_production_auto_validate

LoisRForgeFlow committed:
[IMP] mrp_multi_level_estimate: Estimates and other demand sources strategy

Introduces a new concept to define the strategy to apply when estimates
coexist with other sources of demand.

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_multi_level_estimate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_multi_level_estimate/it/

LoisRForgeFlow committed:
[REW] mrp_multi_level_estimate: remove unneeded dependency

Remove unneeded dependency to date_range and relicense to LGPL.

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_bom_tracking
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_bom_tracking/it/

OCA-git-bot committed:
account_move_line_mrp_info 14.0.1.1.0

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_purchase_link/it/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-14.0/manufacture-14.0-account_move_line_mrp_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-account_move_line_mrp_info/it/

lbelorgey committed:
[MIG] mrp_bom_note: Migration to 14.0

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_sale_info/it/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_bom_tracking
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_bom_tracking/it/

Commit:
Add mrp_lot_production_date

oihane committed:
[MIG] mrp_sale_info: Migration to 16.0

grindtildeath committed:
mrp_p_auto_validate: fix auto_validate flag

Relying on onchange makes this value set only if the BOM is set
manually on the Form view, what isn't the case when MO are created
by running procurements.

grindtildeath committed:
mrp_p_auto_validate: split qty on create

- split values for create in dedicated function
- post messages on MO with modified qty
- split MOs on create only on procurements

Commit:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mrp_multi_level_estimate 15.0.1.1.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-15.0/manufacture-15.0-mrp_multi_level_estimate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_multi_level_estimate/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: manufacture-15.0/manufacture-15.0-mrp_multi_level_estimate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-15-0/manufacture-15-0-mrp_multi_level_estimate/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_purchase_link/it/

Commit:
[ADD] mrp_package_propagation

JuMiSanAr committed:
[MIG] mrp_lot_production_date: Migration to 16.0

ThiagoMForgeFlow committed:
[14.0][FIX] mrp_subcontracting_purchase_link: link to PO line, not PO

JordiBForgeFlow committed:
[IMP] add migration script

Commit:
[UPD] Update mrp_subcontracting_purchase_link.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
mrp_subcontracting_purchase_link 14.0.2.0.0

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-14.0/manufacture-14.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_purchase_link/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (11 of 11 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_subcontracting_purchase_link/it/

JordiBForgeFlow committed:
[IMP] mrp_subcontracting_purchase_link: add post init hook to populate the PO line on previous manufacturing orders

AungKoKoLin1997 committed:
[MIG] mrp_subcontracting_purchase_link: Migration to 16.0

AungKoKoLin1997 committed:
[ADD] migration folder

Urvisha-OSI committed:
[IMP] mrp_bom_tracking: pre-commit stuff

Urvisha-OSI committed:
[MIG] mrp_bom_tracking: Migration to 16.0

Urvisha-OSI committed:
[FIX] resolve codepatch fails issue

AungKoKoLin1997 committed:
[UPD] remove unnecessary condition

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_sale_info/it/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_sale_info/it/

florian-dacosta committed:
[IMP] mrp_sale_info : Allow new fields to be shown or hidden by user to avoid overcomplicating the tree view

BernatPForgeFlow committed:
[IMP] mrp_multi_level_estimate: Stock moves from MO should always be considered

When we consider forecasts as the 'Indirect' demand, we should always keep considering the demand coming from Manufacturing Orders, as they are 'Direct' demand.

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_bom_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_bom_note/it/

ChrisOForgeFlow committed:
[16.0][MIG] account_move_line_mrp_info

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (13 of 13 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_sale_info/es/

AlexPForgeFlow committed:
[MIG] mrp_multi_level_estimate: Migration to 16.0

Commit:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
[UPD] README.rst

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_bom_tracking
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_bom_tracking/it/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (19 of 19 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_bom_tracking
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_bom_tracking/es/

OCA-git-bot committed:
[UPD] README.rst

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (11 of 11 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_multi_level_estimate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_multi_level_estimate/es/

Commit:
[UPD] Update mrp_production_auto_validate.pot

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] icon.png

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_production_auto_validate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_production_auto_validate/it/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-16.0/manufacture-16.0-mrp_multi_level_estimate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_multi_level_estimate/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-14.0/manufacture-14.0-mrp_production_auto_validate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_production_auto_validate/

francesco-ooops committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_production_auto_validate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_production_auto_validate/it/

mymage committed:
Added translation using Weblate (Italian)

Commit:
[UPD] Update mrp_subcontracting_purchase_link.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-16.0/manufacture-16.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_subcontracting_purchase_link/

Commit:
[UPD] Update account_move_line_mrp_info.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-16.0/manufacture-16.0-account_move_line_mrp_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-account_move_line_mrp_info/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (16 of 16 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_production_auto_validate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_production_auto_validate/it/

Commit:
[UPD] Update mrp_lot_production_date.pot

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_lot_production_date
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_lot_production_date/it/

OCA-git-bot committed:
[BOT] post-merge updates

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (1 of 1 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_lot_production_date
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_lot_production_date/it/

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (16 of 16 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_production_auto_validate
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_production_auto_validate/es/

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (5 of 5 strings)

Translation: manufacture-14.0/manufacture-14.0-mrp_bom_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-14-0/manufacture-14-0-mrp_bom_note/es/

Commit:
[IMP] mrp_sale_info: pre-commit auto fixes

Commit:
[MIG] mrp_sale_info: Migration to 17.0

AndreuOForgeFlow committed:
[IMP] mrp_bom_tracking: pre-commit auto fixes

AndreuOForgeFlow committed:
[MIG] mrp_bom_tracking: Migration to 17.0

CarlosVForgeFlow committed:
[IMP] account_move_line_mrp_info: pre-commit auto fixes

CarlosVForgeFlow committed:
[MIG] account_move_line_mrp_info: Migration to 17.0

Minor fixes

Adjustments

Ivorra78 committed:
Added translation using Weblate (Spanish)

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (8 of 8 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_subcontracting_purchase_link/es/

quentinDupont committed:
[16.0][ADD] mrp_bom_image

JordiMForgeFlow committed:
[FIX] mrp_multi_level_estimate: adapt domain to search on child locations

OCA-git-bot committed:
[BOT] post-merge updates

Ivorra78 committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (26 of 26 strings)

Translation: manufacture-17.0/manufacture-17.0-mrp_bom_tracking
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-mrp_bom_tracking/es/

Commit:
mrp_package_propagation: propagate from consumable components

As there is no quant created for consumable products, Odoo does not
support the propagation of a destination package from an ancestor move to its
destination move. As such, the consumable component move of a MO has no package
to propagate even if the Pre-PICK transfer put one.
To support this use-case, we try to find a package to propagate from ancestor
moves, allowing consumable products to propagate packages.

JasminSForgeFlow committed:
[IMP] mrp_multi_level_estimate: black, isort, prettier

JasminSForgeFlow committed:
[MIG] mrp_multi_level_estimate: Migration to 17.0

Commit:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
[BOT] post-merge updates

alfredoavanzosc committed:
[MIG] mrp_bom_note: Migration to 16.0

mide-dynapps committed:
Added translation using Weblate (French)

mide-dynapps committed:
Translated using Weblate (French)

Currently translated at 100.0% (8 of 8 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_subcontracting_purchase_link
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_subcontracting_purchase_link/fr/

SottomaiorMacedoTec committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_bom_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_bom_note/pt_BR/

mymage committed:
Added translation using Weblate (Italian)

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (5 of 5 strings)

Translation: manufacture-16.0/manufacture-16.0-mrp_bom_image
Translate-URL: https://translation.odoo-community.org/projects/manufacture-16-0/manufacture-16-0-mrp_bom_image/it/

mathieudelva committed:
[ADD] notes from bom on report_mrporder

mathieudelva committed:
[UPD] mrp_production_templates

JordiMForgeFlow committed:
[IMP] account_move_line_mrp_info: add MO link in tree view and standard filtering

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (19 of 19 strings)

Translation: manufacture-17.0/manufacture-17.0-mrp_bom_tracking
Translate-URL: https://translation.odoo-community.org/projects/manufacture-17-0/manufacture-17-0-mrp_bom_tracking/it/

legalsylvain committed:
[FIX] mrp_bom_image: use new syntax to filter elements by their classes
So, remove warning during the installation or update of this module

JasminSForgeFlow committed:
[MIG] mrp_multi_level_estimate: Migration to 18.0

BhaveshHeliconia committed:
[IMP] mrp_bom_tracking: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] mrp_bom_tracking: Migration to 18.0

Commit:
[UPD] Update account_move_line_mrp_info.pot

OCA-git-bot committed:
[BOT] post-merge updates

BhaveshHeliconia committed:
[IMP] mrp_sale_info: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] mrp_sale_info: Migration to 18.0

BhaveshHeliconia committed:
[IMP] mrp_bom_image: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] mrp_bom_image: Migration to 18.0

BhaveshHeliconia committed:
[IMP] mrp_bom_note: pre-commit auto fixes

BhaveshHeliconia committed:
[MIG] mrp_bom_note: Migration to 18.0

MeritxellAForgeFlow committed:
[IMP] mrp_subcontracting_purchase_link: pre-commit auto fixes

MeritxellAForgeFlow committed:
[MIG] mrp_subcontracting_purchase_link: Migration to 17.0

Standard migration to 17.0

Commit:
[UPD] Update mrp_subcontracting_purchase_link.pot

OCA-git-bot committed:
[BOT] post-merge updates

chaule97 committed:
[IMP] mrp_production_auto_validate: pre-commit auto fixes

Commit:
[IMP] mrp_lot_production_date: pre-commit execution

Commit:
[MIG] mrp_lot_production_date: Migration to 18.0

Kimkhoi3010 committed:
[MIG] mrp_bom_location: Migration to 18.0

ThiagoMForgeFlow committed:
[IMP] account_move_line_mrp_info: pre-commit auto fixes

ThiagoMForgeFlow committed:
[MIG] account_move_line_mrp_info: Migration to 18.0

chaule97 committed:
[MIG] mrp_production_auto_validate: Migration to 18.0

grindtildeath committed:
[IMP] mrp_package_propagation: pre-commit auto fixes

grindtildeath committed:
[MIG] mrp_package_propagation: Migration to 18.0

OCA-git-bot committed:
[IMP] update dotfiles

LucasTran380381 commented on mrp_production_auto_validate/views/mrp_production.xml:
```diff
@@ -8,7 +8,7 @@
         <field name="arch" type="xml">
             <div name="button_box" position="before">
-                <field name="auto_validate" invisible="1" />
+                <field name="auto_validate" invisible="1" readonly="state != 'draft'" />
```
if it invisible field maybe not need readonly attr. check if invisible field is needed if not we could remove it

LucasTran380381 reviewed (approved):
work as expected 
![image](https://github.com/user-attachments/assets/3c01670d-9d76-4b37-acd2-125d85cb9494)

RicardCForgeFlow committed:
Revert [FIX] mrp_multi_level: adapt tests to recent upstream changes.
After https://github.com/odoo/odoo/commit/f869630fc97db0db95ae5d0f103907bc73ac8ddc
we can revert https://github.com/OCA/manufacture/commit/d84964c0952707b379e1c12c7fec50fe1468709b

OCA-git-bot committed:
Merge PR #1468 into 18.0

Signed-off-by LoisRForgeFlow

OCA-git-bot committed:
[BOT] post-merge updates

xaviedoanhduy commented on mrp_production_auto_validate/readme/CREDITS.md:
```diff
@@ -0,0 +1,3 @@
+The development of this module has been financially supported by:
+
+- Camptocamp
```
```suggestion
The development of this module has been financially supported by:

- Camptocamp
```
looks weird

xaviedoanhduy commented on mrp_production_auto_validate/models/mrp_production.py:
```diff
@@ -0,0 +1,199 @@
+
+import logging
+
+from odoo import api, exceptions, fields, models, tools
```
```suggestion
from odoo import api, exceptions, fields, models
from odoo.tools import float_compare
```

xaviedoanhduy commented on mrp_production_auto_validate/models/mrp_production.py:
```diff
@@ -0,0 +1,199 @@
+                )
+
+                qty_ok = (
+                    tools.float_compare(
```
```suggestion
                    float_compare(
```

OCA-git-bot committed:
Merge PR #1466 into 18.0

Signed-off-by grindtildeath

Commit:
[UPD] Update mrp_package_propagation.pot

OCA-git-bot committed:
[BOT] post-merge updates

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: manufacture-18.0/manufacture-18.0-mrp_package_propagation
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_package_propagation/

mymage committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (17 of 17 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_package_propagation
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_package_propagation/it/

Kimkhoi3010 committed:
[ADD] mrp_bom_location: Add tests

ISNEYLER committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_production_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_production_note/es/

OCA-git-bot committed:
Merge PR #1510 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update mrp_lot_production_date.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1412 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_multi_level_estimate.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1461 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update account_move_line_mrp_info.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1433 into 18.0

Signed-off-by quentinDupont

Commit:
[UPD] Update mrp_bom_image.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1434 into 18.0

Signed-off-by quentinDupont

Commit:
[UPD] Update mrp_bom_note.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1428 into 18.0

Signed-off-by quentinDupont

Commit:
[UPD] Update mrp_sale_info.pot

OCA-git-bot committed:
[BOT] post-merge updates

JordiMForgeFlow committed:
[MIG] mrp_subcontracting_purchase_link: migration to V18

OCA-git-bot committed:
Merge PR #1470 into 18.0

Signed-off-by simahawk

Commit:
[UPD] Update mrp_bom_location.pot

OCA-git-bot committed:
[BOT] post-merge updates

OCA-git-bot committed:
Merge PR #1516 into 18.0

Signed-off-by LoisRForgeFlow

Commit:
[UPD] Update mrp_subcontracting_purchase_link.pot

OCA-git-bot committed:
[BOT] post-merge updates

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Added translation using Weblate (Dutch (nl_NL))

bosd committed:
Translated using Weblate (Dutch (nl_NL))

Currently translated at 100.0% (10 of 10 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_sale_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_sale_info/nl_NL/

bosd committed:
Translated using Weblate (Dutch (nl_NL))

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_bom_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_bom_note/nl_NL/

bosd committed:
Translated using Weblate (Dutch (nl_NL))

Currently translated at 100.0% (2 of 2 strings)

Translation: manufacture-18.0/manufacture-18.0-mrp_production_note
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-mrp_production_note/nl_NL/

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (7 of 7 strings)

Translation: manufacture-18.0/manufacture-18.0-account_move_line_mrp_info
Translate-URL: https://translation.odoo-community.org/projects/manufacture-18-0/manufacture-18-0-account_move_line_mrp_info/nl/

OCA-git-bot committed:
Merge PR #1418 into 18.0

Signed-off-by rvalyi

Commit:
[UPD] Update mrp_bom_tracking.pot

OCA-git-bot committed:
[BOT] post-merge updates
